docs/ehrql-tutorial.md

```diff
@@ -2,7 +2,50 @@
 
 ---8<-- 'includes/data-builder-danger-header.md'
 
-This page will contain an [ehrQL](ehrql-intro.md) tutorial. It will
-initially describe how a relatively minimal ehrQL example works.
+In this tutorial, we will walk through a minimal ehrQL example.
+The example will create a dataset of all patients who were born in or before 2000.
+We will walk through the example step-by-step;
+if you want to see the example in full, then you can [skip to the end](#wrapping-up).
+
+First, we import the patients table and create a variable that references each patient's year of birth:
+
+---8<-- 'examples/src-minimal-ehrql-import-patients.md'
+
+Next, we import and create a dataset
+
+---8<-- 'examples/src-minimal-ehrql-import-dataset.md'
+
+and set the population, which is all patients who were born in or before 2000:
+
+---8<-- 'examples/src-minimal-ehrql-set-population.md'
+
+At this point, we have created a dataset and set the population, but we haven't extracted any data.
```

Comment:
Technically, we have: `patient_id`. Sometimes, I think it's worth trading accuracy for clarity 🙂